packages/types/script/get-proto.sh

```diff
@@ -35,12 +35,12 @@ mkdir $PROTO_DIR
 
 # Mv the proto file into the proto dir
 mv "$TMP_DIR/desmos-$DESMOS_VERSION/proto" $PROTO_DIR
-mv "$TMP_DIR/desmos-$DESMOS_VERSION/third_party" $PROTO_DIR
 
-# Remove all the old proto folders
-rm -r "$PROTO_DIR/proto/desmos/profiles/v1beta1"
-rm -r "$PROTO_DIR/proto/desmos/profiles/v2"
-rm -r "$PROTO_DIR/proto/desmos/subspaces/v1"
+# Install needed third party proto files by telescope
+yarn telescope install @protobufs/cosmos @protobufs/cosmos_proto @protobufs/ibc
+
+# Mv the proto files downloaded by telescope into the proto dir
+find "proto" -mindepth 1 -maxdepth 1 -type d ! -name "desmjs" -exec mv -t "$PROTO_DIR/proto/" {} +
```

Comment:
`telescope` can only download the proto files to `$CURR_DIR/proto`, so I use the command to move them into `$PROTO_DIR`, make `proto` only includes `desmjs` proto files.